src/components/Core/ColoredChip.vue

```diff
@@ -21,18 +21,20 @@ const props = withDefaults(
 const { t } = useI18nUtils()
 const { current } = useTheme()
 const { enableHashColors, hideColoredChip } = storeToRefs(useVueTorrentStore())
+const vueTorrentStore = useVueTorrentStore()
```

Comment:
Prevent duplicate calls
```suggestion
const vueTorrentStore = useVueTorrentStore()
const { enableHashColors, hideColoredChip } = storeToRefs(vueTorrentStore)
```